client/src/pages/Feed.js

```diff
@@ -18,21 +30,151 @@ import {
 // ICONS
 import SvgIcon from "components/Icon/SvgIcon";
 import creatPost from "assets/icons/create-post.svg";
+import { ReactComponent as FiltersIcon } from "assets/icons/filters.svg";
 
 export const FeedContext = React.createContext();
 
+const { Content, Sider } = Layout;
+
+// feed types
+const HELP_TYPE = {
+  ALL: 'All posts',
+  REQUEST: 'Requesting help',
+  OFFER: 'Offering help'
+};
+
 const initialState = {
   filterModal: false,
   createPostModal: false,
   activePanel: null,
   location: "",
 };
 
+const SiderWrapper = styled(Sider)`
```

Comment:
When I scroll down, at top where the space where the nav bar is looks off. We also need to give the Feed a background color of `WHITE`. This will probably make empty space not as apparent.

<img width="654" alt="Screen Shot 2020-05-10 at 8 31 27 PM" src="https://user-images.githubusercontent.com/42228386/81521814-3a41e680-92fd-11ea-9dcf-00270fced7c9.png">

Reply:
Done.